sqlite/src/analyze.c

```diff
@@ -1238,16 +1218,15 @@ static void analyzeOneTable(
   int iDb;                     /* Index of database containing pTab */
```

Comment:
Big simplification of the 'analyze_empty_tables' tunable code in here, please review.